pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -7,6 +7,46 @@
             "#import": [ "base" ]
         },
 
+        "android": {
+            "#import": [ "any" ]
+        },
+        "android-arm64": {
+	           "#import": [ "any" ]
+        },
+        "android-mips": {
+            "#import": [ "any" ]
+        },
+
+        "android.21": {
+	           "#import": [ "android" ]
+	       },
+        "android.21-arm64": {
+	           "#import": [ "android.21", "android-arm64" ]
+        },
+        "android.21-mips": {
+	           "#import": [ "android.21", "android-mips" ]
+        },
+
+        "android.22": {
```

Comment:
Similarly, no work has been done for API level (versions) other than 21, so I'm not sure it makes sense to add RIDs for those versions new.